packages/eslint-config/package.json

```diff
@@ -14,12 +14,12 @@
     "react"
   ],
   "dependencies": {
-    "eslint-config-airbnb": "^17.0.0",
-    "eslint-config-datarockets-base": "latest"
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-datarockets-base": "https://gitpkg.now.sh/datarockets/style-guide/packages/eslint-config-base?upgrade-eslint-config-dependencies"
```

Comment:
Would it make sense if we split out another PR with an independent upgrade of `eslint-config-datarockets-base`, merge it first and then use the `latest` version for the sake of better subsequence?